File: grading-students/grading-students.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> gradingStudents(vector<int> grades) {
    for (int i = 0; i < grades.size(); i++) {
        if (grades[i] >= 38) {
            int x = grades[i] % 5;
            if (x >= 3) {
                grades[i] += (5 - x);
            }
        }
    }

    return grades;
}

File: grading-students/grading-students.py
from typing import List

def gradingStudents(grades: List[int]) -> List[int]:
    for i in range(0, len(grades)):
        if grades[i] >= 38:
            x = grades[i] % 5
            if x >= 3:
                grades[i] += (5 - x)

    return grades